---
format: 
  pdf: 
    documentclass: article
    papersize: A4
    geometry:
      - top=10mm
      - left=15mm
      - right=15mm
      - bottom=5mm
      - heightrounded
    fontfamily: libertine
    colorlinks: true
    fontsize: '16'
    include-in-header: 
      text: |
        \usepackage{multicol}
        \usepackage{lipsum}
        \usepackage{hyperref}
        \usepackage{libertine}
        \pagestyle{empty}
        \setlength{\multicolsep}{2pt plus 1.0pt minus 0.75pt}% 50% of original values
        \setlength{\columnsep}{3em}
---
<!-- TITLE -->
\begin{huge}\begin{center}{\bf Andrew Morris, PhD}\end{center}\end{huge}

<!-- CONTACT INFO -->
\begin{center}andrewmorris@mailbox.org ∙ \href{https://ahmorris.org}{Personal Website} ∙ \href{https://github.com/amorris28}{Github} ∙ \href{https://www.linkedin.com/in/andrew-morris-71033a41/}{LinkedIn}
\end{center}
\begin{center}Located in Groton, Connecticut
\end{center}
<!-- \vspace{15pt} -->

<!-- EDUCATION: 1ST BULLET -->
\begin{large}
  {\bf EDUCATION}
  \vspace{3pt}
  \hrule
  
  {\bf PhD Biology}, (2022) {\it University of Oregon} \hfill Eugene, OR
  
\end{large}

<!-- EDUCATION: 2ND BULLET -->
\begin{large}

    {\bf MS Soil Science}, (2017) {\it The Pennsylvania State University} \hfill State College, PA
    
\end{large}

<!-- EDUCATION: 3RD BULLET -->
\begin{large}

{\bf BS Plant Sciences}, (2014) {\it Cornell University} \hfill Ithaca, NY

\end{large}

\vspace{3pt}

<!-- SKILLS -->
\begin{large}{\bf SKILLS}
  \vspace{3pt}
  \hrule
\end{large}
**Languages and Tools**: R, Bash, Git, Python, GNU Make, Linux

**Selected Coursework**: 
\vspace{-0.16cm}

<!-- * \textit{Machine Learning for Image Analysis} - Introduction to Deep Neural Networks using Keras and Tensorflow -->
* \textit{Advanced Biological Statistics I \& II} - Core concepts and methods in frequentist and Bayesian analysis using R and Stan
* \textit{Strategies and Techniques for Analyzing Microbial Community Population Structures} - Bioinformatics and statistical training to practitioners of molecular microbial ecology and genomics

<!-- EXPERIENCE : 1ST BULLET-->
\begin{large}{\bf EXPERIENCE}
  \vspace{3pt}
  \hrule
  
  {\bf \href{https://www.uoregon.edu}{University of Oregon}} \hfill Aug 2017-Present
  
  \vspace{-0.17cm}
  
  \textit{Post-doctoral Scholar (Mar 2022-Present)} \hfill Eugene, OR, USA
  
  \vspace{-0.17cm}

  \textit{NSF Graduate Research Fellow (Aug 2017-Mar 2022)}
  
\end{large}

* Published multiple scientific papers in peer-reviewed journals
* Authored and co-authored funded grant proposals including a $3 million (USD) award from NSF
* Analyzed data from next-generation sequencing (NGS) technologies including Illumina shotgun metagenomics and amplicon sequencing
* Developed bioinformatics pipelines using sequencing databases and software including BLAST, Bowtie, and SAMtools
* Performed quality control, error correction, mapping, assembly, and annotation on short read sequencing data
* Presented research results to diverse audiences including the general public, industry partners, and scientific specialists
* Mentored early career scientists in data analysis with R as well as written and oral communication
* Maintained data pipelines using GNU make, git, and slurm in a HPC cloud computing environment 

\vspace{7pt}

<!-- EXPERIENCE : 2ND BULLET -->
\begin{large}

  {\bf \href{https://www.psu.edu/}{The Pennsylvania State University}} \hfill Aug 2015-Jul 2017

  \vspace{-0.17cm}
  
  \textit{Graduate Research Assistant} \hfill State College, PA, USA
  
\end{large}

* Conducted industry-partnered experiments with interdisciplinary research teams
* Delivered data analysis results that guided on-farm practices to balance profitability with environmental impacts using sustainable agriculture
* Presented research to farmers, industry partners, and scientists at farmer advisory board meetings, on-farm field days, and scientific meetings

\vspace{7pt}

<!-- EXPERIENCE : 3RD BULLET -->
\begin{large}
  \begin{multicols}{2}
    \begin{flushleft}{\bf \href{https://www.udel.edu/}{University of Delaware}}\end{flushleft}
    \begin{flushright}Feb 2015-Jul 2015\end{flushright}
  \end{multicols}
  \vspace{-0.17cm}
  \begin{multicols}{2}
    \begin{flushleft}\textit{Research Assistant}\end{flushleft}
    \begin{flushright}Newark, DE, USA\end{flushright}
  \end{multicols}
\end{large}
\vspace{-0.16cm}
* Supervised construction and data collection for a field experiment with graduate and undergraduate research assistants
* Managed daily lab work, handled procurement, and contributed to a scientific publication

\vspace{7pt}

<!-- PERSONAL PROJECTS : 1ST BULLET -->
\begin{large}
{\bf PROJECT PORTFOLIO}
  \vspace{3pt}
  \hrule
  
  \textit{\href{https://github.com/amorris28/artificial_ecosystem_selection/}{Prediction of greenhouse gas emissions from soil microbiome composition}} \hfill
    July 2023
    
  \vspace{-0.17cm}
\end{large}
* Conducted an artificial selection experiment to develop microbiomes that perform a high rate of methane consumption.
* Modeled microbial community performance using regression and beta-binomial models.

\vspace{7pt}

<!-- PERSONAL PROJECTS : 2ND BULLET -->
\begin{large}

  \textit{\href{https://github.com/amorris28/microbial_gwas/}{Applying genotype-phenotype mapping to microbial ecosystem functions}} \hfill
    March 2020
    
  \vspace{-0.17cm}
\end{large}
* Demonstrated the use of agnostic search and controlling for data stratification to identify microorganisms associated with important ecosystem functions such as greenhouse gas emissions.

\vspace{7pt}

<!-- PERSONAL PROJECTS : 3RD BULLET -->
\begin{large}
  \textit{\href{https://ahmorris.org/publications/}{Selected Publications}}
  \vspace{-0.17cm}
\end{large}
1.	**Morris AH**, Isbell SA, Saha D, and Kaye JP. 2021. Mitigating nitrogen pollution with undersown legume-grass cover crop mixtures in winter cereals. _Journal of Environmental Quality_ doi: [10.1002/jeq2.20193](https://www.doi.org/10.1002/jeq2.20193)
1.	**Morris AH**, Meyer KM, Bohannan BJM. 2020. Linking microbial communities to ecosystem functions: what we can learn from genotype-phenotype mapping in organisms. _Philosophical Transactions of the Royal Society B_ doi: [10.1098/rstb.2019.0244](https://www.doi.org/10.1098/rstb.2019.0244)
